### DISCUSSION TITLE
AI not creating fields

### DISCUSSION THREAD

**Discussion by jonbennett65** [Courseplay/Courseplay_FS22#3326] [Q&A]
So i have made the custom field and the ai routes it fine. when i turn on the create field for the plow the ai starts plowing. soon as i leave the tractor the ai continues on but it just drags the plow and not creating the fields. cant figure out if im doing something wrong or its bugged. i have made fields a hand full of times with the AI plowing it for me so i know its doable.

**Comment by jonbennett65** [comment]
![fs22help](https://github.com/Courseplay/Courseplay_FS22/assets/91784750/58bddd19-6567-45c2-ae50-dfe7c5a95793)

**Reply by Tensuko** [reply]
Create field for the plow is active I guess?
We are not doing anything with that setting so there must be something else wrong. I check that later.

**Comment by jonbennett65** [comment]
yea its all active but soon as you leave the tractor it just stops plowing and just drags i appreciate the response tho. love the mode yall are life savers haha

**Reply by Tensuko** [reply]
No Problem...
![grafik](https://github.com/Courseplay/Courseplay_FS22/assets/27344744/3d9b1317-fae3-403f-80ca-af377b4d5932)
logfile?

**Reply by jonbennett65** [reply]
how do i get it

**Reply by jonbennett65** [reply]
so i hoped back on ther server today and was gonna try to get the log you asked for and now the plows are working as intended, maybe it was a server thing and when it restarted today fixed it?

**Reply by jonbennett65** [reply]
it started again ima have to send the log

**Reply by Tensuko** [reply]
Does not sound like a cp problem.
